artichoke-backend/src/class_registry.rs

```diff
@@ -96,7 +67,7 @@ impl ClassRegistry for Artichoke {
         } else {
             return Ok(None);
         };
-        let args = args.iter().map(Value::inner).collect::<Vec<_>>();
+        let args = args.iter().map(Self::Value::inner).collect::<Vec<_>>();
```

Comment:
I like using `Self::Value` in the function signatures to match the trait definition, but in the function body, I'd prefer if we use the concrete type instead of indirecting through the trait.

```suggestion
        let args = args.iter().map(Value::inner).collect::<Vec<_>>();
```